shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -53,6 +55,11 @@ public Options getOptions() {
     return new Options().addOption(RECURSIVE_OPTION);
   }
 
+  private static final Pattern USER_PATTERN = Pattern.compile("(?<user>[a-z_][a-z0-9_-]*)");
+
+  private static final Pattern USER_GROUP_PATTERN =
+      Pattern.compile("(?<user>[a-z_][a-z0-9_-]*):(?<group>[a-z_][a-z0-9_-]*)");
```

Comment:
Done.
Originally I used the regex to match UNIX user names, which is not general enough.